scripts/util.py

```diff
@@ -153,6 +155,66 @@
 ] = lambda self, data: yaml.representer.SafeRepresenter.represent_str(self, str(data))
 
 
+class ApiEndpoint(Enum):
+    COMPUTE = "compute"
+    BQ = "bq"
+    STORAGE = "storage"
+    TPU = "tpu"
+    SECRET = "secret_manager"
+
+
+_ud = "googleapis.com"
+
+
+def universe_domain(refresh: bool = False) -> str:
+    global _ud
+    if refresh:
+        _ud = instance_metadata("attributes/universe_domain")
```

Comment:
instance_metadata is cached, so `refresh` has no effect.

Reply:
Done.